api_app/analyzers_manager/observable_analyzers/mmdb_server.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+import requests
+
+from api_app.analyzers_manager import classes
+from api_app.analyzers_manager.exceptions import (  # AnalyzerConfigurationException
+    AnalyzerRunException,
+)
+from tests.mock_utils import MockUpResponse, if_mock_connections, patch
+
+logger = logging.getLogger(__name__)
+
+
+class MmdbServer(classes.ObservableAnalyzer):
+    base_url = "https://ip.circl.lu/geolookup/"
```

Comment:
please configure this attribute to be a valid parameter (the user should be able to change it) https://intelowl.readthedocs.io/en/latest/Contribute.html#example-how-to-add-a-new-parameter-in-the-configuration-with-a-default-value

Reply:
noted